apisix/core/table.lua

```diff
@@ -25,13 +25,14 @@ local type         = type
 
 
 local _M = {
-    version = 0.1,
+    version = 0.2,
     new     = new_tab,
     clear   = require("table.clear"),
     nkeys   = nkeys,
     insert  = table.insert,
     concat  = table.concat,
     clone   = require("table.clone"),
+    isarray = require("table.isarray"),
```

Comment:
why add this line?

Reply:
https://github.com/apache/incubator-apisix/pull/1440/files/2a2da03a7b71ad3314a89253fa478182fa66c22f#diff-e3e08c15cae0b9924d16c46ac8f64230R345

https://github.com/apache/incubator-apisix/pull/1440/files/2a2da03a7b71ad3314a89253fa478182fa66c22f#diff-c96e5aa165284ada8a16fee994fec508R54

https://github.com/apache/incubator-apisix/pull/1440/files/2a2da03a7b71ad3314a89253fa478182fa66c22f#diff-5e074f24e0cf98f1f68e6f4390a49164R38

Check that table is a array.